src/eradius.app.src

```diff
@@ -15,96 +15,7 @@
       {counter_aggregator, false},
       {logfile, "./radius.log"},
       {metrics, [{enabled, [server, nas, client]}]},
-      {recbuf, 8192} % maximum socket receive buffer in bytes
-
-      %% RADIUS server configuration:
```

Comment:
Is this information still available somewhere?

Reply:
Yep, moved to https://github.com/travelping/eradius#radius-server-configuration section. All information in this comments was updated and moved into [README.md](https://github.com/travelping/eradius/blob/master/README.md) in the scope of fixing issue https://github.com/travelping/eradius/issues/165